tests/test_audio.py

```diff
@@ -25,25 +25,22 @@
 
 import telegram
 from tests.base import BaseTest, timeout
-from tests.bots import get_bot
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
```

Comment:
To support python 2 (which we do) you need to use `super(TestAudio, self).setUpClass(cls)` (see Travis' logs)
Same for all others